google/protobuf/duration.proto

```diff
@@ -0,0 +1,104 @@
+// Protocol Buffers - Google's data interchange format
```

Comment:
What are other options other than adding the protobuf protos here? Protos in this repo will be cloned/branched by developers, so we don't want to add temporary/incomplete protos here.